src/pyodide/internal/setupPackages.ts

```diff
@@ -14,12 +13,12 @@ const canonicalizeNameRegex = /[-_.]+/g;
  * @returns The canonicalize package name.
  * @private
  */
-function canonicalizePackageName(name) {
+function canonicalizePackageName(name: string): string {
   return name.replace(canonicalizeNameRegex, "-").toLowerCase();
 }
 
 // The "name" field in the lockfile is not canonicalized
-const STDLIB_PACKAGES = Object.values(LOCKFILE.packages)
+const STDLIB_PACKAGES: string[] = Object.values(LOCKFILE.packages)
```

Comment:
You'll want to add a file `pyodide-internal:generated/pyodide-lock.json.d.ts` to give the types for `LOCKFILE`.

Reply:
Right, I had that, and I forgot to git add it 😪